src/pages/MerchantVoucherDashboardV2/Banners.tsx

```diff
@@ -0,0 +1,53 @@
+import classNames from 'classnames';
+import React from 'react';
+import CheckCircleIcon from '@material-ui/icons/CheckCircle';
+import ErrorIcon from '@material-ui/icons/Error';
+
+import styles from './styles.module.scss';
+
+export const ERROR_TYPE = {
+  GENERIC: 'GENERIC',
+  VALIDATION: 'VALIDATION',
+};
+
+type ErrorTypeKeys = keyof typeof ERROR_TYPE;
+export type ErrorTypeValues = typeof ERROR_TYPE[ErrorTypeKeys];
+
+const ERROR_COPY_ENGLISH = {
+  GENERIC:
+    'There was an error saving your changes. Please refresh your page and try again.',
+  // Right now, the only validation error we have on the update voucher
+  // endpoint is if the amount is higher than the current voucher value.
```

Comment:
if you want, you can make these tsdoc comments with `/** ... */` on either/or the `VALIDATION` property and/or the parent object itself.

Reply:
oh right, updated this code